packages/node/src/plugins/segmentio/publisher.ts

```diff
@@ -198,6 +198,14 @@ export class Publisher {
           return batch.resolveEvents()
         }
 
+        const currentTime = new Date()
```

Comment:
We just want sentAt at the root, we don't need to append this field into every individual event. The integrations pipeline takes care of that.